keps/sig-auth/3766-referential-authorization/README.md

```diff
@@ -413,80 +422,115 @@ rejected".
 
 ### API Spec
 
+#### ClusterReferenceConsumer
+
 ```golang
-// ReferenceGrant identifies kinds of resources in other namespaces that are
-// trusted to reference the specified kinds of resources in the same namespace
-// as the policy.
-//
-// Each ReferenceGrant can be used to represent a unique trust relationship.
-// Additional ReferenceGrants can be used to add to the set of trusted
-// sources of inbound references for the namespace they are defined within.
-//
-// ReferenceGrant is a form of runtime verification allowing users to assert
-// which cross-namespace object references are permitted. Implementations that
-// support ReferenceGrant MUST NOT permit cross-namespace references which have
-// no grant, and MUST respond to the removal of a grant by revoking the access
-// that the grant allowed. 
-//
-// Implementation of ReferenceGrant is eventually consistent, dependent on
-// watch events being received from the Kubernetes API. Although some processing
-// delay is inevitable, any updates that could result in revocation of access MUST
-// be considered high priority and handled as quickly as possible.
-//
-// Implementations of ReferenceGrant MUST treat all of the following scenarios
-// as equivalent:
-//
-// * A reference to a Namespace that doesn't exist
-// * A reference to a Namespace that exists and a Resource that doesn't exist
-// * A reference to Namespace and Resource that exists but a ReferenceGrant
-//   allowing the reference does not exist
-//
-// If any of the above occur, a generic error message such as "RefNotPermitted" 
-// should be communicated, likely via status on the referring resource.
-//
-// Support: Core
-type ReferenceGrant struct {
-	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
+// Subject refers to the subject that is a consumer of the referenced
+// pattern(s).
```

Comment:
```suggestion
// ClusterReferenceConsumer authorizes the given subject to access a referenced resource
// in the same namespace.
```

Reply:
Could probably word this better still.

Reply:
Consumer resources do not authorize.
They opt-in to matching Grants.

These comments do need to be cleaned up, because it looks like they are stale.